src/parser/csql_lexer.l

```diff
@@ -132,7 +140,12 @@ int is_dblink_query_string = 0;
 [aA][rR][iI][aA]							{ begin_token(yytext);   
                                                                                 csql_yylval.cptr = pt_makename(yytext);
                                                                                 return ARIA; }
-[aA][sS]								{ begin_token(yytext);   return AS; }
+[aA][sS]								{ begin_token(yytext);
+                                                                                if (expecting_pl_lang_spec) {
+                                                                                        expecting_pl_lang_spec = 0;
+                                                                                        BEGIN(PL_LANG_SPEC);
+                                                                                }
+                                                                                return AS; }
```

Comment:
`AS` can be used when specifying column alias.
```
SELECT a AS 'alias' FROM tbl;
```
Is it ok to move to PL_LANG_SPEC mode without any consideration?

Reply:
It happens when expecting_pl_lang_spec flag is on. 
The flag is turned on by the parser when CREATE (OR REPALCE) PROCEDURE/FUNCTION tokens are found in this order.

Reply:
Anyway, I realized that the flag is not turned off for a statement having a syntax error of no IS or AS after the CREATE (OR REPALCE) PROCEDURE/FUNCTION token stream. I will fix it.